pkg/client/logs.go

```diff
@@ -228,8 +229,43 @@ type logStreamer struct {
 	client             kubernetes.Interface
 }
 
+func isContainerRunning(statuses *[]v1.ContainerStatus, containerName string) bool {
+	for _, cs := range *statuses {
+
+		fmt.Printf("container %v, state= %v\n", containerName, cs.State)
```

Comment:
No need to have this print each time we check fi something is running.

Reply:
stale debug log line. removed :)